GVFS/GVFS.UnitTests/Maintenance/GitMaintenanceStepTests.cs

```diff
@@ -72,11 +72,13 @@ public CheckMethodStep(GVFSContext context)
 
             protected override void PerformMaintenance()
             {
-                this.RunGitCommand(process =>
-                {
-                    this.SawWorkInvoked = true;
-                    return null;
-                });
+                this.RunGitCommand(

```

Comment:
Note: This comment is really for line 29, but I can't comment there ☹️ 

If any of the unit tests in this class will now hit `throw new StoppingException();` in the product code they should be tagged with `[Category(CategoryConstants.ExceptionExpected)]`

Reply:
Since the exception is catch (never leaks to the test) I don't I should add the tag right?

Reply:
We've been tagging unit tests with `ExceptionExpected` when any of the code that runs in the test throws an exception.

We started doing that because running the unit tests with the debugger attached became an exercise in pressing continue to get through all of the expected exceptions.

The `ExceptionExpected` just prevents the test from being run by default when the debugger is attached.